source/server/lds_api.cc

```diff
@@ -75,13 +76,16 @@ void LdsApiImpl::onConfigUpdate(const ResourceVector& resources, const std::stri
         ENVOY_LOG(debug, "lds: add/update listener '{}' skipped", listener_name);
       }
     } catch (const EnvoyException& e) {
-      throw EnvoyException(
-          fmt::format("Error adding/updating listener {}: {}", listener_name, e.what()));
+      exception_msgs.push_back(fmt::format("{}: {}", listener_name, e.what()));
```

Comment:
This doesn't capture the duplicate listener case above, do we also want to support continuation for that?

Reply:
This I tried to make it consistent with how CDS behaves. If there are duplicates it just rejects the message with out adding any cluster - which I think may be OK.

Reply:
Fair enough.